app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -53,13 +53,13 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         }
 
         override fun onSessionRemoved(session: Session) {
-            // There's no active session. Start a new session with "homepage".
-            ScreenController.showBrowserScreenForUrl(this@MainActivity, supportFragmentManager, APP_URL_HOME, Source.NONE)
+            if (components.sessionManager.sessions.isEmpty()) {
```

Comment:
The current logic is:
```kotlin
if (sessionsIsEmpty) 
    showBrowserScreenForHome()
else
    showBrowserScreenForCurrentUrl()
```

The logic in this method isn't the same. Is the logic the same because `onSessionSelected` will be called if `onSessionRemoved` is called? If so, this is good; if not, please fix it!

Reply:
>  Is the logic the same because onSessionSelected will be called if onSessionRemoved is called? 

Yes! If the removed session was the selected session then `SessionManager` will determine the next selected session (like in Fennec) and will notify observers via `onSessionSelected()`.